### ISSUE TITLE
Compilation failed during e2e because of core-js

### ISSUE THREAD

**Issue by creal73** [ngx-rocket/generator-ngx-rocket#465]
## I'm submitting a...

- [x] Bug report  
- [ ] Feature request
- [ ] Documentation issue or request

## Current behavior

Compilation failed running e2e tests

![image](https://user-images.githubusercontent.com/587284/57130227-1e763d00-6d99-11e9-9074-c25d870d3a3c.png)

## Expected behavior

Compilation succeeded

## Minimal reproduction of the problem with instructions

Working on `feature/jest` branch, I'm running e2e tests on a generated project.

## Environment

```
- generator version: 6.0.1
- node version: 10.15.3
- npm version: 6.4.1
- OS: Ubuntu 19.04
```

**Comment by creal73** [comment]
Regarding [this post](https://github.com/angular/angular-cli/issues/13954#issuecomment-485822832), I suggest to roll bak to `core-js@2.x`

**Comment by creal73** [comment]
Closed by #466
